### ISSUE TITLE
Have exception if playing Chinese song

### ISSUE THREAD

**Issue by tyutony** [JohnnyCrazy/SpotifyAPI-NET#171]
`StatusResponse status = _spotify.GetStatus()`
these have a exception when run this code if  Spotify playing Chinese song.

> Newtonsoft.Json.JsonReaderException ( "After parsing a value an unexpected character was encountered: u. Path '[0].track.artist_resource.name', line 20, position 5.")

And if song is Chinese song and firing OnTrackChange event , it will have a exception and event stop firing.

> "Newtonsoft.Json.JsonReaderException" ("After parsing a value an unexpected character was encountered: u. Path '[0].track.track_resource.name', line 13, position 5.")

**Comment by JohnnyCrazy** [comment]
Link to the chinese song?

**Comment by tyutony** [comment]
Almost all the Chinese song have this problem.
https://open.spotify.com/track/1erIu51Xu1cMckHBoRaika
https://open.spotify.com/track/0187cGIAXfBN9aX4mY79Vg
https://open.spotify.com/track/0VqSdtXseb9khdZrnYVyM1
Some link for you that you can test.

**Comment by JohnnyCrazy** [comment]
Just tested this with the example app, works fine.

![alt](http://i.imgur.com/YBfQBJd.png)

Maybe some encoding issue on your side? Maybe spotify sends a weird JSON, try inspecting the whole payload.

**Comment by tyutony** [comment]
Here is my test on the example app in this video,
In video i just download project and work it when my Spotify client app stop on the Chinese song.
As you can see it really have excepiton.
Here is the video:[https://youtu.be/Z32-zN5H0II?t=40s](https://youtu.be/Z32-zN5H0II?t=40s)
Maybe it happen on the use different language  PC or use different language Spotify Client?

**Comment by tyutony** [comment]
Hi , i think i fixed this excepiton.
the problem is on the `SpotifyAPI.Local.RemoteHandler.Query()`.
Becaouse `WebClient.DownloadString()` this function in Chinese language well get weird code.
So i add this line in `SpotifyAPI.Local.RemoteHandler.Query()` to conversion code to UTF8.
```C# 
wc.Encoding = Encoding.UTF8;
```
And becaouse we aleardy  conversion code to UTF8 so SpotifyAPI.Local.RemoteHandler.GetNewStatus() can delete some part(here i just comment out).
```C#
// byte[] bytes = Encoding.Default.GetBytes(response);
// response = Encoding.UTF8.GetString(bytes);
```
And the whole be edited two function is here.
Maybe you can test this change.

SpotifyAPI.Local.RemoteHandler.Query()
```C#
internal StatusResponse GetNewStatus()
        {
            string response = Query("remote/status.json", true, true, -1);
            if (string.IsNullOrEmpty(response))
            {
                return null;
            }
            response = response.Replace("\\n", "");
            // byte[] bytes = Encoding.Default.GetBytes(response);
            // response = Encoding.UTF8.GetString(bytes);
            List<StatusResponse> raw = JsonConvert.DeserializeObject<List<StatusResponse>>(response);
            return raw[0];
        }
 ```

SpotifyAPI.Local.RemoteHandler.GetNewStatus()
 ```C#
internal string Query(string request, bool oauth, bool cfid, int wait)
        {
            string parameters = "?&ref=&cors=&_=" + GetTimestamp();
            if (request.Contains("?"))
            {
                parameters = parameters.Substring(1);
            }

            if (oauth)
            {
                parameters += "&oauth=" + OauthKey;
            }
            if (cfid)
            {
                parameters += "&csrf=" + CfidKey;
            }

            if (wait != -1)
            {
                parameters += "&returnafter=" + wait;
                parameters += "&returnon=login%2Clogout%2Cplay%2Cpause%2Cerror%2Cap";
            }

            string address = "https://" + Host + ":4371/" + request + parameters;
            string response = "";
            try
            {
                using (var wc = new ExtendedWebClient())
                {
                    if (SpotifyLocalAPI.IsSpotifyRunning())
                        wc.Encoding = Encoding.UTF8;   // just add this line
                        response = "[ " + wc.DownloadString(address) + " ]";
                }
            }
            catch
            {
                return string.Empty;
            }

            return response;
        }
```

**Comment by JohnnyCrazy** [comment]
@tyutony 
Nice findings, would you mind opening a PR? 👍 
I guess you could add the encoding directly to `ExtendedWebClient`

**Comment by JohnnyCrazy** [comment]
Fixed by #174
